masterlist.yaml

```diff
@@ -4662,14 +4662,21 @@ plugins:
         util: 'TES4Edit v4.0.4b'
       - crc: 0x07E94F26
         util: 'TES4Edit v4.0.4b'
-  - name: 'OCO Cleaned - Vanilla Hair.esp'
+  - name: '(OCO Cleaned - Vanilla Hair|OCO Vanilla Hair)\.esp'
     url:
       - link: 'https://www.nexusmods.com/oblivion/mods/51324/'
         name: 'Oblivion Character Overhaul Cleaned - OCO Cleaned'
+      - link: 'https://www.nexusmods.com/oblivion/mods/51998/'
+        name: 'OCO Vanilla Hair - Version FR'
     msg:
       - <<: *wildEdits
         subs: [ 'Remove **0001D9D8**, **000234B7**, & **00034E7E** from **OCO Cleaned - Vanilla Hair.esp**.' ]
         condition: 'checksum("OCO Cleaned - Vanilla Hair.esp", CF743DF7)'
+    clean:
+      - crc: 0xB1E49BA0
+        util: 'TES4Edit v4.0.4b'
+      - crc: 0x3291418D
+        util: 'TES4Edit v4.0.4b'
```

Comment:
`OCO Vanilla Hair.esp` is hosted on a separate page that also doesn't host `OCO Cleaned - Vanilla Hair.esp`, so it'd be more appropriate to separate these plugins. You can place the new plugin just after `OCO uses merged teeth.esp` in the masterlist (i.e., alphanumerically sorted), making sure there's a single extra newline between the two and another after it to keep it properly separated.

The new plugin would look like this:

```yaml
  - name: 'OCO Vanilla Hair.esp'
    url:
      - link: 'https://www.nexusmods.com/oblivion/mods/51998/'
        name: 'OCO Vanilla Hair - Version FR'
    clean:
      - crc: 0xB1E49BA0
        util: 'TES4Edit v4.0.4b'
      - crc: 0x3291418D
        util: 'TES4Edit v4.0.4b'
```

Reply:
Hello Sibir,

I understand. I'm sorry I'm still learning. I will make the necessary changes. Thanks for your help.

Reply:
No worries, that's what the review process is for. You're welcome, & thank you for contributing!

Reply:
I made the changes, hoping I didn't make a mistake.